src/main/java/org/k8loud/executor/drools/DroolsService.java

```diff
@@ -84,35 +84,24 @@ private StatelessKieSession createSession() {
         return kieSession;
     }
 
+    @Async
```

Comment:
is async needed?

Reply:
https://www.baeldung.com/spring-scheduled-tasks#schedule-a-task-at-a-fixed-rate
```
Note that scheduled tasks don’t run in parallel by default. 
So even if we used fixedRate, the next task won’t be invoked until the previous one is done.
If we want to support parallel behavior in scheduled tasks, we need to add the @Async annotation
```